### PR TITLE
Footer and Governors Modifcation

### PR DESCRIPTION
**Pull request by Praneeth-rdy**
## Files Changed
1) Markdown files in societies directory
2) Footer's scss and component

### CODE REVIEW

File: styles/_footer.scss [hunk 1]
```diff
@@ -3,6 +3,8 @@
   padding: 1rem 2rem;
   margin: 2rem;
   min-width: 80vw;
+  flex-direction: row;
+  justify-content: center;
 
   padding: 1rem;
```
Comment (sahil-shubham) [inline styles/_footer.scss:9]: There are two padding properties, we can remove the padding altogether as it's creating a width issue in mobile view.

File: styles/_footer.scss [hunk 2]
```diff
@@ -15,8 +17,20 @@
     display: flex;
     justify-content: space-between;
 
-    li {
+    .footer-block{
+      margin-right: 4vw;
```
Comment (sahil-shubham) [inline styles/_footer.scss:21]: better to define margin in terms of `rem`

File: societies/swg.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ The Student Mentorship Programme is the flagship initiative of SWG which is held
 - Enabling constructive and positive interaction, guidance and mentorship of junior students by senior students. 
 - Providing a reliable and comprehensive support system from within and for the student community to motivate students to excel in both academic and non-academic fields and to make the most of their life at IIT Kharagpur. 
 
-Governors: <br />
```
Comment (sahil-shubham) [inline societies/swg.md:15]: Could you remove the image from this file. We will add it again when we have some.

**Review by sahil-shubham — CHANGES_REQUESTED** [review]
I have left some comments, everything else looks good. Thanks for this PR.

**Comment by Praneeth-rdy** [comment]
I have made the necessary changes. Check the last commit once.

**Review by sahil-shubham — APPROVED** [review]
Looks good to me, there is some padding issue but I will fix it in future PRs. Thanks.

**Pull request merged**